### PR TITLE
PROBLEM: Repo (agent) doesn't work on container

### PR DESCRIPTION
**Pull request by karolhrdina**
SOLUTION: Fix it!

### CODE REVIEW

File: src/fty-metric-compute.service.in [hunk 1]
```diff
@@ -2,16 +2,14 @@
...
 
 [Service]
 Type=simple
 User=bios
-EnvironmentFile=-/usr/share/bios/etc/default/bios
-EnvironmentFile=-/usr/share/bios/etc/default/bios__%n.conf
-EnvironmentFile=-/etc/default/bios
-EnvironmentFile=-/etc/default/bios__%n.conf
-EnvironmentFile=-/etc/default/bios-db-rw
+EnvironmentFile=-/usr/share/fty/etc/default/fty
```
Comment (jimklimov) [inline src/fty-metric-compute.service.in]: Are you sure you want to ignore legacy file/path-names for configs?

File: src/fty_metric_compute.c [hunk 2]
```diff
@@ -79,7 +79,7 @@ int main (int argc, char *argv [])
     zstr_sendx (cm_server, "TYPES", "min", "max", "arithmetic_mean", NULL);
     zstr_sendx (cm_server, "STEPS", "15m", "30m", "1h", "8h", "24h", "7d", "30d", NULL);
     // TODO: Make this configurable, runtime and build-time default
-    zstr_sendx (cm_server, "DIR", "/var/lib/bios/bios-agent-cm/", NULL);
+    zstr_sendx (cm_server, "DIR", "/var/lib/fty/fty-metric-compute/", NULL);
```
Comment (jimklimov) [inline src/fty_metric_compute.c]: I was told to not rename such dirs... so what is the rule of thumb?

File: src/fty_metric_compute.conf.in [hunk 3]
```diff
@@ -1,3 +0,0 @@
-# create runtime directories for bios-agent-cm
```
Comment (jimklimov) [inline src/fty_metric_compute.conf.in]: Why is this removed? The .in should stay, and the generated file (above) should be dropped. And also note the paths (legacy or new)?

Comment (jimklimov) [inline src/fty_metric_compute.conf.in]: Ok, now I saw the individual commit with reasoning. I think it does make sense in a short-term solution to make things "just work" quickly, but is BS regarding longterm FOSS distribution management, where datadir etc. are configurable and templatable (no hardcoded paths in committed sources, ability to install proto area under DESTDIR, etc).

**Comment by jimklimov** [comment]
@karolhrdina : note, many of my questions are related to https://github.com/42ity/fty-nut/pull/8 as an "opposite" direction from yours here regarding pathnames that I understood should be taken. So plz comment - which pattern should be followed?

**Comment by vyskocilm** [comment]
Hi, please return the old names for state files. For configuration it's not a big deal as we have symlinks all around the system.

So /var/lib/bios/bios-agent-cm/ or /var/lib/fty/bios-agent-cm/ are fine, otherwise you need to add compat link to preinstall.sh script ...
